From 810800ac8456d18bb66a3d2e1643ab46570aa388 Mon Sep 17 00:00:00 2001
From: Daniel Ammann <daniel.ammann@bytesatwork.ch>
Date: Wed, 2 Mar 2022 16:18:17 +0100
Subject: [PATCH] arm64: dts: freescale: imx8mm-byteengine: Fix PMIC int

Under certain unknown circumstances, the kernel would not boot properly.
---
 arch/arm64/boot/dts/freescale/imx8mm-byteengine.dtsi | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8mm-byteengine.dtsi b/arch/arm64/boot/dts/freescale/imx8mm-byteengine.dtsi
index 27240ec357969..24d1f158a0fdc 100644
--- a/arch/arm64/boot/dts/freescale/imx8mm-byteengine.dtsi
+++ b/arch/arm64/boot/dts/freescale/imx8mm-byteengine.dtsi
@@ -34,7 +34,7 @@
 
 	pinctrl_pmic: pmicirq {
 		fsl,pins = <
-			MX8MM_IOMUXC_SD1_RESET_B_GPIO2_IO10	0x41	/* R23  SD1_RESET_B  GPIO2_IO10 */
+			MX8MM_IOMUXC_GPIO1_IO03_GPIO1_IO3	0x41	/* AF13  GPIO1_IO03  GPIO1_IO03 */
 		>;
 	};
 
@@ -125,7 +125,7 @@
 
 		pinctrl-names = "default";
 		pinctrl-0 = <&pinctrl_pmic>;
-		gpio_intr = <&gpio2 10 GPIO_ACTIVE_LOW>;
+		gpio_intr = <&gpio1 3 GPIO_ACTIVE_LOW>;
 
 		gpo {
 			nxp,drv = <0x0C>;	/* 0b0000_1100 all gpos with cmos output mode */
-- 
2.20.1

